framework/src/meshgenerators/BreakMeshByBlockGenerator.C

```diff
@@ -171,27 +219,86 @@ BreakMeshByBlockGenerator::addInterfaceBoundary(MeshBase & mesh)
 {
   BoundaryInfo & boundary_info = mesh.get_boundary_info();
 
-  boundary_id_type boundaryID = findFreeBoundaryId(mesh);
-  std::string boundaryName = _interface_name;
+  boundary_id_type boundaryID;
+  boundary_id_type boundaryID_interface = Moose::INVALID_BOUNDARY_ID;
+  boundary_id_type boundaryID_interface_transition = Moose::INVALID_BOUNDARY_ID;
```

Comment:
To remain consistent, I'd prefer that we use `boundary_id` like we do in other places. We typically don't have stuff like "boundaryid" in a single phrase unless it's a type, i.e., `BoundaryID`

```suggestion
  auto boundary_id = Moose::INVALID_BOUNDARY_ID;
  auto boundary_id_interface = Moose::INVALID_BOUNDARY_ID;
  auto boundary_id_interface_transition = Moose::INVALID_BOUNDARY_ID;
```